### DISCUSSION TITLE
Checking for a Python interpreter with version >= 3... none

### DISCUSSION THREAD

**Discussion by AltriaLu** [Homebrew/discussions#3164] [Getting started] [answered]
When installing a dependency called 'fontconfig' of mpv, it reports this error.But actually I have installed python3.10. How to solve this problems?

**Comment by bvahdat** [comment, marked as answer]
I think this PR is relevant: https://github.com/Homebrew/homebrew-core/pull/98756

**Reply by AltriaLu** [reply]
yeah actually  I solve this problem by using interactive mode of brew install and make the configuration myself then this problem no longer exists

**Reply by alps2006** [reply]
I want to know how to solve this problem.  what is your configuration ?

**Reply by AltriaLu** [reply]
Just assign the  python intepreter path as a input parameter of the .configure file. When you look into the .configure, you can find one parameter called 'PYTHON_PATH' or etc. Then run this .configure with your python path assigned and problems could be solved.

**Reply by alps2006** [reply]
Thanks, I passed the compilation.
But i used interactive mode and executed with original configure parameters.

```shell
➜  Downloads> brew install -i fontconfig
➜  fontconfig-2.14.0> ./configure \
> --disable-dependency-tracking \
> --disable-silent-rules \
> --disable-docs \
> --enable-static \
> --with-add-fonts=/system/library/fonts,/library/fonts,~/library/fonts,/system/library/assets/com_apple_mobileasset_fon
t5 \
> --prefix=/usr/local/cellar/fontconfig/2.14.0 \
> --localstatedir=/usr/local/var \
> --sysconfdir=/usr/local/etc
➜  fontconfig-2.14.0> make
➜  fontconfig-2.14.0> make install
➜  fontconfig-2.14.0> exit
```
